src/components/InputValidationMessage/InputValidationMessage.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Used by form inputs such as TextInput, to display a validation message for an invalid input.
+ */
+
+const InputValidationMessage = ({
+  children,
+}) => (
+  <div className="font-color-danger font-size-sm" style={{ marginTop: 'var(--size-spacing-xs)' }}>{children}</div>
```

Comment:
There is no utility class yet for margin and padding from design tokens, so I'm using the css var so I we don't have to have another css file just for this.

Reply:
@nathanyoung good call, y will create these.